### ISSUE TITLE
[Improvement]: Web console default deployment to the running one

### ISSUE THREAD

**Issue by ffguven** [shuttle-hq/shuttle#1853]
### Describe the improvement

Hi folks,

I have a project named `zort-2`, and it contains 4 deployments

* 1 `Stopped` and `Dirty`.
* 2 `Stopped` and `Committed`.
* 1 `Running` and `Committed`.

When I open the Web Console to see logs, it defaults to the first deployment shown in the list although it's stopped and not begin used anymore. That might be misleading when an operator wants to check the logs for the running version of the deployment, or at least requires the operator to manually pick the running deployment all the time.

Instead, either

1. Sorting the deployments with the order `Running` deployment and then the `Stopped` deployments by their timestamp.
2. Showing the active deployment's pane by default.

can be done to improve that part.

IMHO, having the 1st option is more useful in terms of the developer experience and reduce the cognitive overhead when debugging/observing a system.

### Duplicate declaration

- [X] I have searched the issues and this improvement has not been requested before.

**Comment by oddgrd** [comment]
I agree, showing logs for a stopped deployment when you have a running deployment is not useful, and it's unexpected. The console isn't open-source yet, but we'll look into this, thanks for the report and the suggested solutions!

**Comment by jonaro00** [comment]
For console.shuttle.dev, no deployment logs are opened by default. If we add that, we'll reconsider the logic for which one to pick.
